### ISSUE TITLE
[QA: primary wagtail pages] remove indent on image component 

### ISSUE THREAD

**Issue by kristinashu** [mozilla/foundation.mozilla.org#1355]
Images on https://foundation.mozilla.org/wagtail/about/leadership/ are slightly to the right of everything else. The edge of the photo should lineup with the logo, titles, text and everything else.

![screen-shot-2018-04-13-at-3 31 15-pm](https://user-images.githubusercontent.com/1068783/38760450-2bac30f8-3f30-11e8-8ce7-a02043236a60.jpg)

**Comment by alanmoo** [comment]
This is part of a bigger conversation that should be happening this week around components and layout. cc @gvn

**Comment by kristinashu** [comment]
Looks like this is an issue for image with a caption but good for regular images https://foundation.mozilla.org/wagtail/opportunity/tech-policy-fellows/

<img width="740" alt="screen shot 2018-04-17 at 10 26 37 am" src="https://user-images.githubusercontent.com/1068783/38886205-1b8f5c8e-422a-11e8-9417-926fddc76b5d.png">

**Comment by Pomax** [comment]
the figure component currently has bootstrap margins on it so that figuregrid doesn't just glue them together. I don't know if there's bootstrap rules around giving something margins only if it's inside another thing... /cc @gvn

**Comment by kristinashu** [comment]
@gvn is it helpful to tackle this now or wait till we open new tickets based on #1438?

I only notice the intent on the `Figure` component, I think you've already fixed it everywhere else.

**Comment by gvn** [comment]
Yeah I think it's only still on standalone Figures. Are those still used?

**Comment by kristinashu** [comment]
Figures are currently [in use on prod](https://foundation.mozilla.org/opportunity/global-sprint/tools-working-open/) and I believe in the future Figures will map to `Large Image` (Full width).

**Comment by gvn** [comment]
In that case let's tackle in the upcoming tickets.
